resources/assets/javascripts/admin.js

```diff
@@ -110,7 +112,7 @@ jQuery(function() {
     $.get(
       ajaxUrl + '?action=catfishimporter_get_status',
       function onResponse(response) {
-        console.log(response)
+        console.log(response);
```

Comment:
Ah with our JS we don't add semicolons. It's not a biggy though as we haven't really discussed JS coding style.